envoy/network/listener.h

```diff
@@ -150,6 +150,11 @@ class ListenerConfig {
    */
   virtual ListenSocketFactory& listenSocketFactory() PURE;
```

Comment:
Sorry also if the plan is to remove this function can you add a TODO here also?

Reply:
yea, will do, thanks